### ISSUE TITLE
Windows Powershell Lagging while typing in windows terminal

### ISSUE THREAD

**Issue by Rasesh2005** [microsoft/terminal#6400]
# Expected behavior

Should Be Responsive As i type in any CLI Interface.

# Actual behavior

When I complete typing my whole command and look at the screen it has started typing The first word.Sowly typing letter by letter.

I don't have any such problem on opening powershell from win + x But I love the design Of terminal,Therefore came here to report the issue.

By the way, Command Prompt(cmd.exe) is working absolutely fine but i am more comfortable with interface of windows powershell.

Please Fix this bug as soon as possible

I Dont Have any Recording Tool To Attach The Recording.So,sorry couldn't give proof

Summing up, The problem is only Windows Powershell is lagging a lot while typing in windows terminal.But runs Perfectly Fine While opened as Standalone(form win + x )

**Comment by oising** [comment]
@Rasesh2005 -- can you please reply with the following information:
 
- windows version (e.g. windows 10 --  patch 1803/1903/1909/2004 etc.)
- graphics card (e.g. intel iris, nvidia gfx 960 etc etc) and the driver version
- screen resolution (e.g. 4000 x 3000) and screen refresh rate (e.g. 60hz?)
- laptop screen or external monitor? both? Does the lag happen on both screens or just external?
- what version of windows powershell? Try running it as `powershell.exe -noprofile`  does it still lag?

**Comment by Rasesh2005** [comment]
windows version -- 2004 
amd radeon graphics chip(don't remember the exact model)
720p display
laptop

$psversiontable is as follows

Name                           Value
----                           -----
PSVersion                      5.1.19041.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

powershell.exe -noprofile  doesnt make any diffference

But My concern is that it is working smoothly when launched from win+x or right clicking start button. But When Opened in terminal it lags a lot,like after typing the whole sentence, its still typing  first word at the rate of one letter every 2 seconds

**Comment by skyline75489** [comment]
When launched from Win+X that's the original conhost which uses GDI+. Windows Terminal is something very different. It uses DirectX and DirectWrite technology. It also adds a lot of other fancy things. So it should be expected that Windows Terminal has different performance comparing to conhost. Of course we do not want the lagging. Just to point out the essential difference of those two fellows.

**Comment by Rasesh2005** [comment]
Ohk! Thanks For this Valuable Knowledge. Didn't know this.  But I am frustrated by windows terminal but want it to be better because of its better customisations ... SomeOne Plz Fix this issue ASAP

**Comment by skyline75489** [comment]
We'd also love to fix whatever problem you're experiencing. But you gotta give us more context. The CPU & GPU, memory, driver, all those things may help. 

By the way have you tried [software rendering](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/global-settings#software-rendering)? This may also help.

**Comment by vadimkantorov** [comment]
( related feature request about measuring time-to-character-rendering: https://github.com/microsoft/terminal/issues/5590 )

**Comment by Rasesh2005** [comment]
you can check the feature of hp pavillion g6 1117tx notebook pc 
I didn't make any modifications over it

**Comment by oising** [comment]
@Rasesh2005 Try what @skyline75489  mentions above -- there's a setting you can change that will use a different method to draw the screen. It may alleviate the problem by bypassing bugs in your video drivers.

**Comment by oising** [comment]
For ref, his machine has a core i3 cpu and AMD Radeon HD 6470M for gpu.

**Comment by Rasesh2005** [comment]
Can I get help where to paste the settings? Because I don't have much idea about  how json settings work

**Comment by zadjii-msft** [comment]
Sure! You want to add it to the top-level of the json object, next to `defaultProfile`, like so:
![image](https://user-images.githubusercontent.com/18356694/84146641-9cd3f280-aa21-11ea-8016-f1c51e321acb.png)

**Comment by Rasesh2005** [comment]
Sometime when I kill it from task manager (though it ends itself when exit.but doesn’t fix the issue) , It gets Fixed and I am able to use it smoothly

**Comment by oising** [comment]
@Rasesh2005 Is this a work computer? Do you have a corporate antivirus running? Or your own personal one?

**Comment by Rasesh2005** [comment]
personal Computer. So No Urgency
